### ISSUE TITLE
Random failure in Unit.ApparentHorizons.StrahlkorperGr.AreaElement

### ISSUE THREAD

**Issue by nilsdeppe** [sxs-collaboration/spectre#1663]
# Bug reports:

```
-------------------------------------------------------------------------------
Unit.ApparentHorizons.StrahlkorperGr.AreaElement
-------------------------------------------------------------------------------
/work/spectre/tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp:838
...............................................................................
/work/spectre/tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp:504: FAILED:
  CHECK( integral_1 == approx(integral_2) )
with expansion:
  0.5035009167 == Approx( 0.5035009167 )
with message:
  Seed is: 942835398
0.124 s: Unit.ApparentHorizons.StrahlkorperGr.AreaElement
===============================================================================
test cases:    1 |    0 passed | 1 failed
assertions: 1362 | 1361 passed | 1 failed
```

In https://travis-ci.org/sxs-collaboration/spectre/jobs/569841856

**Comment by nilsdeppe** [comment]
Here's another seed that failed: `3723524747` on https://travis-ci.org/sxs-collaboration/spectre/jobs/574026184

**Comment by wthrowe** [comment]
```
-------------------------------------------------------------------------------
Unit.ApparentHorizons.StrahlkorperGr.AreaElement
-------------------------------------------------------------------------------
/home/wthrowe/spectre/tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp:838
...............................................................................

/home/wthrowe/spectre/tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp:504: FAILED:
  CHECK( integral_1 == approx(integral_2) )
with expansion:
  -0.7600075794 == Approx( -0.7600075794 )
with message:
  Seed is: 664513909
```

**Comment by wthrowe** [comment]
```
-------------------------------------------------------------------------------
Unit.ApparentHorizons.StrahlkorperGr.AreaElement
-------------------------------------------------------------------------------
/home/wthrowe/spectre/tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp:838
...............................................................................

/home/wthrowe/spectre/tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp:504: FAILED:
  CHECK( integral_1 == approx(integral_2) )
with expansion:
  -1.5283497878 == Approx( -1.5283497878 )
with message:
  Seed is: 1058902770
```

**Comment by nilsdeppe** [comment]
Here's another case:
```
-------------------------------------------------------------------------------
Unit.ApparentHorizons.StrahlkorperGr.AreaElement
-------------------------------------------------------------------------------
/work/spectre/tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp:838
...............................................................................
/work/spectre/tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp:504: FAILED:
  CHECK( integral_1 == approx(integral_2) )
with expansion:
  0.3293970448 == Approx( 0.3293970448 )
with message:
  Seed is: 3365882729
```

**Comment by geoffrey4444** [comment]
Last failure was in September 2019. My proposed fix got merged in October 2019 (PR #1702), no reported failures since. I ran the test overnight repeatedly, with no failures. I pushed the last known reported seed to fail to GitHub, and the tests still passed.

I propose that we close this issue.

**Comment by geoffrey4444** [comment]
Fixed with #1702 ... please reopen if new failure cases appear
